lib/chef/azure/commands/enable.rb

```diff
@@ -66,9 +66,7 @@ def enable_chef
     begin
       configure_chef_only_once
 
-      install_chef_service if @exit_code == 0
-
-      enable_chef_service if @exit_code == 0
+      enable_chef_service

```

Comment:
@mwrock sorry..my bad..I thought the conditional `if @exit_code == 0` previously written was not making any sense as at first it is initialized to `0` in `initialize` method of the `ChefEnable` class and later on its value is changed inside `enable_chef_service` method. But I missed checking `handler_settings_file` method where its value is manipulated. I will add the conditional again.

Regarding `settings file`, we have this method `handler_settings_file` where error handling is done if the file is not present and this same method is invoked inside the `configure_chef_only_once` and `enable_chef_service` method which will raise the error.